multiqc/utils/util_functions.py

```diff
@@ -99,6 +90,23 @@ def default(self, obj):
 
                 body = "\n".join(rows)
 
+            except:
+                data_format = "yaml"
+                config.logger.warning(
+                    fn
+                    + " could not be saved as tab-separated output. Choose JSON or YAML output instead. Falling back to YAML."
+                )
```

Comment:
```suggestion
               logger.debug(f"{fn} could not be saved as tsv/csv. Falling back to YAML.")
```

Reply:
I stand corrected! `config.logger` wasn't a mistake I see but intentional! Smart. Logging in this file has always been awkward (see other function where I pass the logger as an argument). This seems like quite an elegant solution though..